pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
Its slightly difficult to grasp what each of these options stand for. I can make out `-e` is for in-place. But what about `-p` and `-i` ?

We usually aim to have atleast a word starting with the character of the option, or highlight the character mid-word li[k]e this. Can we do something like this here ?